### ISSUE TITLE
URL Preview GitHub/GitLab file URLs with code snippets

### ISSUE THREAD

**Issue by t3chguy** [vector-im/element-meta#1295]

**Comment by daniellekirkwood** [comment]
Do we have more info or an example of the suggestion here?

**Comment by t3chguy** [comment]
Like this `https://github.com/element-hq/element-meta/blob/1a93f254dcb1b55435fb29a99d83f9b8f3a411c8/package.json#L10-L12`

https://github.com/element-hq/element-meta/blob/1a93f254dcb1b55435fb29a99d83f9b8f3a411c8/package.json#L10-L12

**Comment by richvdh** [comment]
This might be better filed as a feature request against Synapse?

**Comment by daniellekirkwood** [comment]
i don't think it should be in Ele-Meta -- I still don't track what's being asked for, the problem, impact or expected outcome (any one of these would be fine)
